miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/FileWriter.kt

```diff
@@ -1,3 +1,19 @@
 package com.rakuten.tech.mobile.miniapp.storage
 
-internal class FileWriter
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.io.File
+import java.io.InputStream
+
+internal class FileWriter {
+
+    suspend fun write(inputStream: InputStream, path: String) {
+        withContext(Dispatchers.IO) {
+            File(path).mkdirs()
+            val file = File(path)
+            inputStream.use { it.copyTo(file.outputStream()) }
+            inputStream.close()
```

Comment:
I think calling `inputStream.use` will automatically close the InputStream after it finishes executing the block, doesn't it?

Reply:
That's correct. Took care of it. Thanks.